### PR TITLE
Categories styling

### PR DESCRIPTION
**Pull request by marcterre**

### CODE REVIEW

File: components/Form.js [hunk 1]
```diff
@@ -22,39 +27,145 @@ export default function Form({ handleSubmit }) {
...
+
+const Input = styled.input`
+  ${styledInputs}
+  min-height: 5vh;
+  &:last-of-type {
+    border: none;
+    background: none;
+    font-size: 0.9em;
+  }
+`;
```
Comment (mbosselmann) [inline components/Form.js:92]: Why did you decide to not put these stylings in one styled component? 🤔

File: components/Form.js [hunk 2]
```diff
@@ -22,39 +27,145 @@ export default function Form({ handleSubmit }) {
...
+    background-color: var(--color-buttons-yellow);
+    padding: 0.2em 3.5em;
+  }
+  &:first-of-type {
+    background-color: var(--color-list-items-white);
+    color: var(--color-project-inactive);
+    fill: var(--color-project-inactive);
+    padding: 0.2em 3em;
+  }
 `;
```
Comment (mbosselmann) [inline components/Form.js:143]: I think you may have buttons with almost the same styling. 🤔 You could think about creating a styled button component with different variants to reuse some of your buttons. See our tourio app to get inspiration: https://github.com/mbosselmann/tourio/blob/main/components/StyledButton.js

File: components/Navigation.js [hunk 3]
```diff
@@ -1,38 +1,80 @@
...
+  border-radius: 50%;
+  background-color: ${({ isFocused }) =>
+    isFocused === "/create-new-project"
+      ? "var( --color-background)"
+      : "var(--color-buttons-yellow)"};
+  &:active {
+    position: relative;
+    top: 1px;
+  }
+`;
```
Comment (mbosselmann) [inline components/Navigation.js:62]: I would also recommend to work with one IconWrapper. See the art gallery example above to reduce your code. :)

File: components/Navigation.js [hunk 4]
```diff
@@ -1,38 +1,80 @@
...
+              <AddIcon />
+            </AddIconWrapper>
+          </Link>
+        </li>
+      </List>
+    </StyledNavigation>
   );
 }
 
+const HomeIconWrapper = styled.div`
```
Comment (mbosselmann) [inline components/Navigation.js:36]: To use a div inside of an anchor element is not the solution I would prefer. [See next.js documentation on how to style a next/link with styled-components.](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag) But you can also style the link directly. [See art gallery app navigation.](https://github.com/mbosselmann/art-gallery-app/blob/main/components/Navigation/index.js)

File: styles.js [hunk 5]
```diff
@@ -1,6 +1,25 @@
...
+  --color-project-active: #2C9E45;
+  --color-project-inactive: #B20E0E;
+  --color-status-active: #55D940;
+  --color-status-inactive: #D94040;
+
+--border-yellow: 0.2em solid var(--color-buttons-yellow);
+  --border-darkblue: 0.2em solid var(--color-darkblue);
+
+}
+
```
Comment (mbosselmann) [inline styles.js:22]: Cool! 👍

**Review by mbosselmann — COMMENTED** [review]
Hi Marc,
great job! 👍 The review of your code took me longer than I thought as there many changes in your code (as you know). 👀
In general, I had the impression that you often use similar stylings that differ only slightly. Here you should think about how you can improve the styling by setting variants or referencing existing styled components. See my comments for some ideas. :)

File: components/Form.js (hunk 1 above)
Comment (marcterre) [reply components/Form.js:92]: because the textarea gets the same styling, i just renamed it and would leave it like that :)

**Comment by marcterre** [comment]
Hey Mareike, thanks for your recommendations, i try to use the variants more often. I changed everything 👍

File: pages/project-details/[id]/index.js [hunk 6]
```diff
@@ -20,6 +23,7 @@ export default function ProjectDetails({
   const [projects, setProjects] = useAtom(projectsAtom);
   const [statusUpload, setStatusUpload] = useAtom(statusUploadAtom);
   const [editImage, setEditImage] = useAtom(showEditImageAtom);
+  const [activeTabbar, setActiveTabbar] = useState(true);
```
Comment (SaschaRadojkovic) [inline pages/project-details/[id]/index.js:26]: maybe change activeTabbar to activeTabBar

File: pages/project-details/[id]/index.js [hunk 7]
```diff
@@ -139,33 +143,55 @@ export default function ProjectDetails({
...
-          </ViewLink>
-          <ViewItem
-            views={views.filter((view) => view.projectId === currentProject.id)}
-            projects={projects}
-            currentProject={currentProject}
-          />
-        </ViewsSection>
-        {currentProject.dimensions && (
+        <Wrapper>
+          <ButtonTabbar onClick={() => setActiveTabbar(true)}>
```
Comment (SaschaRadojkovic) [inline pages/project-details/[id]/index.js:167]: maybe the ButtonTabBar also in camelcase .. but it is only for readability

**Review by SaschaRadojkovic — APPROVED** [review]
left you only some little comments .but approved it anyway . very nice project

**Pull request merged**